File: sample/src/main/java/com/lany192/box/sample/ui/settings/feedback/FeedbackActivity.java
package com.lany192.box.sample.ui.settings.feedback;

import android.os.Bundle;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.github.lany192.arch.activity.ViewBindingActivity;
import com.github.lany192.arch.utils.BarUtils;
import com.lany192.box.sample.databinding.ActivityFeedbackBinding;

import dagger.hilt.android.AndroidEntryPoint;

@AndroidEntryPoint
@Route(path = "/ui/feedback")
public class FeedbackActivity extends ViewBindingActivity<ActivityFeedbackBinding> {

    @Override
    public void initImmersionBar() {
        BarUtils.init(this).keyboardEnable(true).titleBar(binding.toolbar).init();
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding.button.setOnClickListener(v -> {

        });
    }
}

File: sample/src/main/java/com/lany192/box/sample/ui/settings/feedback/FeedbackActivity.kt
package com.lany192.box.sample.ui.settings.feedback

import android.os.Bundle
import com.alibaba.android.arouter.facade.annotation.Route
import com.github.lany192.arch.activity.ViewBindingActivity
import com.github.lany192.extension.toast
import com.github.lany192.extension.addStatusBarPadding
import com.github.lany192.utils.KeyboardWatcher
import com.lany192.box.sample.databinding.ActivityFeedbackBinding
import com.lany192.box.sample.dialog.ReplyDialog
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
@Route(path = "/ui/feedback")
class FeedbackActivity : ViewBindingActivity<ActivityFeedbackBinding>() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        
        binding.toolbar.setTitle("反馈")
        binding.button.setOnClickListener {
            ReplyDialog().show()
        }
        KeyboardWatcher(this, object : KeyboardWatcher.OnKeyboardListener {

            override fun onChanged(showKeyboard: Boolean, keyboardHeight: Int) {
                toast("键盘弹出:$showKeyboard,高度：$keyboardHeight")
            }
        })
    }
}